config/checkstyle_checks.xml

```diff
@@ -137,6 +137,12 @@
     <property name="message"
               value="Old site links should not be used, please use https://checkstyle.org"/>
   </module>
+  <module name="RegexpSingleline">
+    <property name="id" value="noDefaultCommentWithOtherVisibility"/>
+    <property name="format" value="/\* default \*/ (private|protected|public)"/>
```

Comment:
This should be `/\* (default|package) \*/ (private|protected|public)`

The rule [CommentDefaultAccessModifier](https://pmd.github.io/pmd-6.21.0/pmd_rules_java_codestyle.html#commentdefaultaccessmodifier) supports both `/* package */` and `/* default */` comments. We use both of them in the code:

`/* package */ ParseErrorMessage(int lineNumber, String messageKey,`
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java#L727

`/* default */ FollowingEnumeration(NodeInfo start) {`
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/xpath/ElementNode.java#L368

Reply:
We should follow their regular expression exactly, imo, as it allows leeway on spacing.
https://github.com/pmd/pmd/blob/master/pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/codestyle/CommentDefaultAccessModifierRule.java#L42

Reply:
I made pdm rule more strict, please see second commit